addons/project/models/project.py

```diff
@@ -1165,6 +1165,11 @@ def SELF_READABLE_FIELDS(self):
     def SELF_WRITABLE_FIELDS(self):
         return PROJECT_TASK_WRITABLE_FIELDS
 
+    @api.depends('project_id')
+    def _compute_analytic_account_id(self):
+        for task in self:
+            task.analytic_account_id = task.project_id.analytic_account_id
```

Comment:
In the case, where the AA is manually changed than we erase to add the one of the project.
We should change the AA only if the AA has not manually changed by the user and the AA is different than the project one and:
- the project of the task changes
- or the AA of the project changes.

To do that, we should probably add a new Boolean to set to True when the AA in the task is manually changed otherwise set to False.

We do the same for the `cost`s field in the employee mapping.
See here: https://github.com/odoo/odoo/blob/18952cdc76070f2a70789f2fa25e4fa0d546d470/addons/sale_timesheet/models/project_sale_line_employee_map.py#L57-L66

However, we have recently made a fix for that, see this #90332